public/mathlib/transform.h

```diff
@@ -0,0 +1,49 @@
+#ifndef TRANSFORM_H
+#define TRANSFORM_H
+
+#ifdef _WIN32
+#pragma once
+#endif
+
+#include "mathlib/vector.h"
+#include "mathlib/mathlib.h"
+
+class ALIGN16 CTransform
+{
+public:
+	CTransform() {}
+	CTransform( const Vector &v, const Quaternion &q ) : m_vPosition(v), m_orientation(q) {}
+
+	VectorAligned m_vPosition;
+	QuaternionAligned m_orientation;
```

Comment:
Can you move these to the bottom of the class definition? Also I guess ``m_orientation`` is better to be changed to ``m_Orientation`` for naming consistency.

Reply:
Seeing that ``m_orientation`` is also named like that in the dwarf bin I guess nvm.